torch/nn/attention/__init__.py

```diff
@@ -67,6 +60,32 @@ def _raise_kernel_warnings(params: SDPAParams) -> None:
             can_use_flash_attention(params, True)
 
 
+_backend_names = {
```

Comment:
can you leave a note that if this changes the Dynamo variable also needs to be updated

Reply:
The `_backend_list` variable in dynamo doesn't need to be there actually - so no updates to dynamo are necesssary.